jaxlib/cuda/cuda_kernels.cc

```diff
@@ -1,70 +0,0 @@
-/* Copyright 2021 Google LLC
```

Comment:
Please revert this change.

This is actually not a vestigal artifact, but it's not used by JAX itself directly.

It exists for the convenience of some users of JAX who stage out HLO IR from JAX and run it separately from JAX itself. Of course, HLO staged out by JAX may contain calls to these kernels, so this is an additional library that statically links in the necessary CUDA kernels to run JAX HLO without any of the Python mechanisms.

We could add something similar to the ROCm side although I might wait until there's a user asking for it.

Reply:
> Please revert this change.
> 
> This is actually not a vestigal artifact, but it's not used by JAX itself directly.
> 
> It exists for the convenience of some users of JAX who stage out HLO IR from JAX and run it separately from JAX itself. Of course, HLO staged out by JAX may contain calls to these kernels, so this is an additional library that statically links in the necessary CUDA kernels to run JAX HLO without any of the Python mechanisms.
> 
> We could add something similar to the ROCm side although I might wait until there's a user asking for it.

Done.  Sorry about that.